### PR TITLE
CORS policy 

### PR DESCRIPTION
**Pull request by Davidzhu001**
CORS policy: No 'Access-Control-Allow-Origin'

If we run rails on pow, the domain name is different with localhost one.

### CODE REVIEW

File: lib/install/config/webpack/development.server.js [hunk 1]
```diff
@@ -10,6 +10,8 @@ module.exports = merge(devConfig, {
     host: devServer.host,
     port: devServer.port,
     compress: true,
+       headers: { "Access-Control-Allow-Origin": "*" },
+
```
Comment (sekrett) [inline lib/install/config/webpack/development.server.js]: This helped me a lot! That line really should be there. I have problems loading Bootstrap fonts and it worked a few days ago on Rails 5.1.0.rc2, now on 5.1.0 it does not. Please fix and release a new version ASAP.

Comment (Davidzhu001) [inline lib/install/config/webpack/development.server.js]: Thanks for your command.

**Comment by gauravtiwari** [comment]
@Davidzhu001 Could you please drop the template.rb change? It's already there. Just add the cors part.  Also change " to ' (quotes in development.server.js) please

**Comment by Davidzhu001** [comment]
@gauravtiwari how can i drop it?

**Comment by gauravtiwari** [comment]
Just sync your fork with master -

```
git remote add upstream git@github.com:rails/webpacker.git
git pull upstream master
git push
```

That should do it and then push it your fork

**Comment by Davidzhu001** [comment]
@gauravtiwari  Thanks so much

**Comment by gauravtiwari** [comment]
Great 👍  thanks to you too 🍰

**Pull request merged**